resolwe_bio/tests/files/slamseq/alleyoop/hs_alleyoop_summary.txt

```diff
@@ -0,0 +1,3 @@
+# slamdunk summary v0.3.4
+FileName	SampleName	SampleType	SampleTime	Sequenced	Mapped	Deduplicated	MQ-Filtered	Identity-Filtered	NM-Filtered	Multimap-Filtered	Retained	Counted	Annotation
+/data_all/1/hs_slamseq_filtered.bam	sample_0	NA	-1	22	22	0	0	0	0	0	22	22	hs_transcript.bed
```

Comment:
The test for this process will fail whenever multiple tests will be triggered at the same time (the bam input path will not always be `/data_all/1`). To filter out such lines when running `assert` tests, you can use file filters such as [this](https://github.com/genialis/resolwe-bio/blob/1d2bf4d7eda0f91ae4238c89a29e8fb16a9fc0ba/resolwe_bio/tests/workflows/test_chemut.py#L59). If the same filter rule will be used in multiple places, define the filter function [here](https://github.com/genialis/resolwe-bio/blob/1d2bf4d7eda0f91ae4238c89a29e8fb16a9fc0ba/resolwe_bio/utils/filter.py).